src/main/java/com/example/bookshop/dto/order/UpdateStatusDto.java

```diff
@@ -0,0 +1,11 @@
+package com.example.bookshop.dto.order;
+
+import com.example.bookshop.model.Status;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+
+@Data
+public class UpdateStatusDto {
+    @NotBlank
```

Comment:
```suggestion
    @NotNull
```
Because Status is an object